flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -95,7 +95,7 @@ public OperatorStateBackend createOperatorStateBackend(
 			Environment env,
 			String operatorIdentifier) throws Exception {
 
-		return new DefaultOperatorStateBackend(env.getUserClassLoader());
+		return new DefaultOperatorStateBackend(env.getUserClassLoader(), env.getExecutionConfig());
```

Comment:
You can take a look at `DefaultKeyedStateStore`.  Operator states are supposed to have same structures. Otherwise, there exist too many inconsistent code in flink.